pages/common/zip.md

```diff
@@ -33,3 +29,7 @@
 - Archive a directory and its contents to a multi-part [s]plit zip file (e.g. 3GB parts):
 
 `zip -r -s {{3g}} {{compressed.zip}} {{path/to/directory}}`
+
+- Scan and show files within the archive (without processing them):
```

Comment:
:thinking: I think `extract` is a more accurate verb
```suggestion
- List files within the archive (without extract them):
```

Reply:
It is, @navarroaxel, but you've got the word ending slightly wrong. What about this?
```suggestion
- List files within a specified archive (without extracting them):
```